Gulpfile.js

```diff
@@ -122,6 +130,22 @@ gulp.task('spress-build-after-sass', ['sass'], function () {
     .pipe(exec(defaults.spress_bin + ' site:build --source=' + defaults.spress_home));
 });
 
+//Build Spress Production Artifact
+gulp.task('spress-prod', function () {
+  console.log('Building production artifact...');
+  console.log('WARNING: this will overwrite the existing build');
+  return gulp.src(defaults.spress_home)
+    .pipe(exec(defaults.spress_bin + ' site:build --env=prod --source=' + defaults.spress_home));
+});
+
+// Set a deploy task for spress
+gulp.task('spress-deploy', ['spress-prod'], function() {
+  console.log('Beginning deploy to gh-pages for' + defaults.repo);
+  return gulp.src(defaults.spress_output)
```

Comment:
This didn't work for me initially, but when I change defaults.spress_output to defaults.output_prod it did work.